packages/api/tests/lambdas/test-cleanExecutions.js

```diff
@@ -1,58 +1,470 @@
-'use strict';
-
-const sinon = require('sinon');
+/* eslint-disable no-await-in-loop */
 const test = require('ava');
+const moment = require('moment');
+const clone = require('lodash/clone');
+const { randomId } = require('@cumulus/common/test-utils');
+const {
+  translatePostgresExecutionToApiExecution,
+  fakeExecutionRecordFactory,
+  destroyLocalTestDb,
+  generateLocalTestDb,
+  ExecutionPgModel,
+  migrationDir,
+  localStackConnectionEnv,
+} = require('@cumulus/db');
+const { cleanupTestIndex, createTestIndex } = require('@cumulus/es-client/testUtils');
+const { sleep } = require('@cumulus/common');
+const { handler, getExpirationDates } = require('../../lambdas/cleanExecutions');
+test.beforeEach(async (t) => {
+  t.context.testDbName = randomId('cleanExecutions');
+  const { knex, knexAdmin } = await generateLocalTestDb(t.context.testDbName, migrationDir);
 
-const { cleanExecutionPayloads } = require('../../lambdas/cleanExecutions');
-
-const removeRecordsStub = sinon.spy();
+  t.context.knex = knex;
+  t.context.knexAdmin = knexAdmin;
+  const { esIndex, esClient, searchClient } = await createTestIndex();
+  t.context.esIndex = esIndex;
+  t.context.esClient = esClient;
+  t.context.searchClient = searchClient;
+  const records = [];
+  for (let i = 0; i < 10; i += 1) {
```

Comment:
A little redundancy in test setup isn't terrible but there's overlap with another PR that's doing a very similar setup. I wonder if a test helper is in order.

https://github.com/nasa/cumulus/pull/3692/files#diff-723ecf957415d7198e55ff736cc0c60ca586962793d7b18d424bb8ab45dc3ae9R86